mdaviz/utils.py

```diff
@@ -81,19 +80,19 @@ def get_det(mda_file_data):
     The first detector is labeled D01 and subsequent detectors follow in numerical order.
 
     Parameters:
-    - mda_file_data: An instance of an mda.scanDim object, which contains the MDA file data to be processed.
+        - mda_file_data: An instance of an mda.scanDim object, which contains the MDA file data to be processed.
 
     Returns:
-    - A tuple containing:
-      - A dictionary (d) where keys are indexes, mapping to either scanPositioner or scanDetector objects.
-        The dictionary is structured as {0: p0, 1: P1, ..., np: D01, np+1: D02, ..., np+nd: DX}.
-      - The index (first_pos) of the first positioner in the returned dictionary. This is 1 if a positioner
-        other than the default index positioner exists, otherwise 0.
-      - The index (first_det) of the first detector in the returned dictionary, which directly follows the last positioner.
-
-    Note:
-    - p0 is created by default and corresponds to the point index, described as an 'Index' scanPositioner object with predefined properties.
-    - np is the total number of positioners, nd the number of detectors, and npts the number of data points actually acquired.
+        - A tuple containing:
+            - A dictionary (d) where keys are indexes, mapping to either scanPositioner or scanDetector objects.
+                The dictionary is structured as {0: p0, 1: P1, ..., np: D01, np+1: D02, ..., np+nd: DX}.
+            - The index (first_pos) of the first positioner in the returned dictionary. This is 1 if a positioner
+                other than the default index positioner exists, otherwise 0.
+            - The index (first_det) of the first detector in the returned dictionary, which directly follows the last positioner.
+
+    ..note::
```

Comment:
see above about `.. note:: some text` directives

Reply:
Not really sure what I could write there to describe those notes, they are notes lol

Reply:
I have to run out the door, I'll take care of it tomorrow.